dozer-types/src/ingestion_types.rs

```diff
@@ -414,3 +453,104 @@ pub fn default_buffer_size() -> u32 {
 pub fn default_snowflake_poll_interval() -> Duration {
     Duration::from_secs(60)
 }
+
+impl SchemaExample for MongodbConfig {
+    fn example() -> Self {
+        Self {
+            connection_string: "mongodb://localhost:27017/db_name".to_owned(),
+        }
+    }
+}
+
+impl SchemaExample for MySQLConfig {
+    fn example() -> Self {
+        Self {
+            url: "mysql://root:1234@localhost:3306/db_name".to_owned(),
+            server_id: Some((1).to_owned()),
+        }
+    }
+}
+
+impl SchemaExample for GrpcConfig {
+    fn example() -> Self {
+        Self {
+            host: Some("localhost".to_owned()),
+            port: Some(50051),
+            schemas: GrpcConfigSchemas::Path("schema.json".to_owned()),
+            adapter: Some("arrow".to_owned()),
+        }
+    }
+}
+
+impl SchemaExample for KafkaConfig {
+    fn example() -> Self {
+        Self {
+            broker: "{{broker_url}}".to_owned(),
+            schema_registry_url: Some("{{schema_registry_url}}".to_owned()),
```

Comment:
These two are still remaining.

Reply:
Should I put an empty string here, or some default value?

Reply:
just leave an empty value if you are unsure.